tests/test_stackexchange.py

```diff
@@ -0,0 +1,332 @@
+
+        self.assertEqual(raw_questions[0], question)
+        self.assertDictEqual(httpretty.last_request().querystring, payload)
+
+    @httpretty.activate
+    def test_get_questions_from_date(self):
+        """Test question is returned from a given date"""
+
+        question = read_file('data/stackexchange_question')
+
+        httpretty.register_uri(httpretty.GET,
+                               STACKEXCHANGE_QUESTIONS_URL,
+                               body=question, status=200)
+
+        payload = {
+            'page': ['1'],
+            'pagesize': ['1'],
+            'order': ['desc'],
+            'sort': ['activity'],
+            'tagged': ['python'],
+            'site': ['stackoverflow'],
+            'key': ['aaa'],
+            'filter': [QUESTIONS_FILTER],
+            'min': ['1456876800']
+        }
+        # unixtime: 1456876800
+        from_date = datetime.datetime(2016, 3, 2)
+
+        client = StackExchangeClient(site="stackoverflow", tagged="python", token="aaa", max_questions=1)
+        raw_questions = [questions for questions in client.get_questions(from_date=from_date)]
+
+        self.assertEqual(raw_questions[0], question)
+        self.assertDictEqual(httpretty.last_request().querystring, payload)
+
+    @httpretty.activate
+    def test_get_questions_more(self):
+        """Test question API call"""
+
+        page_1 = read_file('data/stackexchange_question_page')
+        page_2 = read_file('data/stackexchange_question_page_2')
+
+        httpretty.register_uri(httpretty.GET,
+                               STACKEXCHANGE_QUESTIONS_URL,
+                               body=page_2,
+                               status=200,
+                               forcing_headers={
+                                    'Link': '<'+STACKEXCHANGE_QUESTIONS_URL+'?&page=2>; rel="next", <'+STACKEXCHANGE_QUESTIONS_URL+'?&page=2>; rel="last"'
+                               })
+        httpretty.register_uri(httpretty.GET,
+                               STACKEXCHANGE_QUESTIONS_URL+'?&page=2',
+                               body=page_1,
+                               status=200)
+        payload = {
+            'page': ['2'],
+            'pagesize': ['1'],
+            'order': ['desc'],
+            'sort': ['activity'],
+            'tagged': ['python'],
+            'site': ['stackoverflow'],
+            'key': ['aaa'],
+            'filter': [QUESTIONS_FILTER]
+        }
+
+        client = StackExchangeClient(site="stackoverflow", tagged="python", token="aaa", max_questions=1)
+        raw_questions = [questions for questions in client.get_questions(from_date=None)]
+
+        self.assertEqual(len(raw_questions), 2)
+        self.assertEqual(raw_questions[0], page_1)
+        self.assertEqual(raw_questions[1], page_2)
+        self.assertDictEqual(httpretty.last_request().querystring, payload)
+
+class TestStackExchangeCommand(unittest.TestCase):
```

Comment:
You have to add a test to check whether the required parameters for this backend are parsed or not.